knuverse/knufactor.py

```diff
@@ -16,8 +16,10 @@
 
 class Knufactor:
     def __init__(self,
-                 apikey,
-                 secret,
+                 apikey=None,
+                 secret=None,
+                 username=None,
```

Comment:
since this can only be used for emails in cloud deployment, might be better to call this variable "email". we could re-evaluate if eventually client login is supported